Data/ByteString/Short/Internal.hs

```diff
@@ -320,8 +320,7 @@ unpackBytes bs = unpackAppendBytesLazy bs []
 -- (5 words per list element, 8 bytes per word, 100 elements = 4000 bytes)
 
 unpackAppendCharsLazy :: ShortByteString -> [Char] -> [Char]
-unpackAppendCharsLazy sbs cs0 =
-    go 0 (length sbs) cs0
+unpackAppendCharsLazy sbs cs0 = go 0 (length sbs) cs0
```

Comment:
personally I prefer the other style, but the style isn't exactly consistent here anyway.